pkg/volume/rbd/attacher.go

```diff
@@ -232,3 +240,30 @@ func (detacher *rbdDetacher) UnmountDevice(deviceMountPath string) error {
 func (detacher *rbdDetacher) Detach(volumeName string, nodeName types.NodeName) error {
 	return nil
 }
+
+func GetDevicePath(plugin *rbdPlugin, deviceMountPath string) string {
+	deviceMountedPaths := strings.Split(deviceMountPath, "-image-")
+	if len(deviceMountedPaths) != 2 {
+		return ""
+	}
+
+	rbdImageName := deviceMountedPaths[1]
+	// rbd showmapped | grep txx9-38-volume-113-huawei | awk '{print $5}'
+	command := "rbd showmapped | grep " + rbdImageName + " | awk '{print $5}'"
```

Comment:
Is it possible to find rbd device in "/sys/bus/rbd/devices" by image name, like https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/rbd/rbd_util.go#L72?